authorities.md

```diff
@@ -14,7 +14,6 @@ Example:
       {
         "id": "srsc",
         "name": "srsc",
-        "scrollable": false,
```

Comment:
this flag was intent to be used by authorities that allow scrolling of all values without providing a filter query. It is needed by the dropdown, the current implementation doesn't use this information but I still see a reason for it (it could detect misconfiguration when a dropdown is used with a not compatibile authority and maybe allow when working in suggest mode to automatically propose suggestion for scrollable authorities without user input when the input receive the focus)

Reply:
@abollini This flag is however not functional at all. It's never set to true, and the dropdowns are currently broken.
Since the feature is not documented, not functional, and has usability issues compared to DSpace 6, it's best to have it removed unless there's an intention to rectify this in the near future.

Reply:
@benbosman : I'm confused why this PR is still modifying the `authorities` REST Contract, as I thought the decision was to change this PR to only create the Read-Only `external-sources` endpoint.